test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -490,7 +493,17 @@ def run(self, test):
         self.result = result = self._makeResult()
         self.result.start = datetime.datetime.now()
         try:
-            test(result)
+            for i in range(self.repetition):
+                self.logger.info("====================================")
+                self.logger.info("tests are running (" + str(i + 1) + ") time")
+                self.logger.info("====================================")
+                if i is not 0:
```

Comment:
I suggest not to use `is not` (py3.8 gives a warning for it), also you can do `if i != 0:` here